src/Attribute/HandlerParameterInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestModel\Attribute;
+
+interface HandlerParameterInterface
```

Comment:
It's not clear about the purpose of it based on the name.

Reply:
Renamed to `HandlerParameterAttributeInterface`